### DISCUSSION TITLE
How to achieve internationalization of multi-version documents?

### DISCUSSION THREAD

**Discussion by misitebao** [facebook/docusaurus#7104] [Q&A] [answered]
I have a multi-language document, but when I execute the `docusaurus docs:version xxx` command, only English documents can display multiple versions. Documents in other languages do not support multiple versions, and the official documentation does not seem to explain what to do.

**Comment by Josh-Cena** [comment, marked as answer]
Hi! Please see https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#translation-files-location

Also, in https://docusaurus.io/docs/i18n/tutorial#translate-markdown-files, there's the mention that:

> Notice that the docusaurus-plugin-content-docs plugin always divides its content by versions.

**Reply by misitebao** [reply]
This means I have to manually create a `versionName` folder in the `i18n/[locale]/docusaurus-plugin-content-docs` directory and move the translated files into it, right? Is there a command like `docusaurus docs:version x.x.x --locale xxx` to autocomplete.

**Reply by Josh-Cena** [reply]
I don't think there is. Would be a useful extension though, would implement that very soon.

**Reply by misitebao** [reply]
Thanks for your help😊

**Reply by Josh-Cena** [reply]
@misitebao See #7106

**Reply by slorber** [reply]
agree there are a few missing DX use-cases in the CLI for i18n 😅 so far nobody complained much (and things can be copied manually) so it got delayed on my todolist

thanks @Josh-Cena

**Reply by xigaoku** [reply]
I test on version 2.0.1 and 2.1.0, this problem is also there! @Josh-Cena
